website/docs/tutorials/08-cairo-1-support/01-understanding-cairo-packages.md

```diff
@@ -127,5 +127,7 @@ You now use your function with `hello_starknet::mod1::functions::returns_three()
 
 ## Packages and modules names considerations
 
-The names must use only ASCII alphanumeric characters or `_`, and cannot be empty. Additionally, it cannot start with an
-underscore.
\ No newline at end of file
+The name must be a valid Cairo identifier. 
+It must use only ASCII alphanumeric characters or underscores, 
+and cannot be empty, nor can it start with a digit.
+It also must not be a valid Cairo keyword or a single underscore (`_`).
```

Comment:
List of constraints would be more readable

Reply:
I tried to make it similar to Scarb's [docs](https://docs.swmansion.com/scarb/docs/reference/manifest#name)

Reply:
I don't think we necessarily have to follow scarb docs style.